recipes/sasctl/recipe.yaml

```
@@ -0,0 +1,50 @@
+context:
+  name: sasctl
+  version: "1.11.1"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/5e/94/8d2c7fd826d0c71c2e567f433c55282e6871230bc7cb244988f329c91fd9/${{ name }}-${{ version }}.tar.gz
+  sha256: 8315d4612304b7afde12c453af0d46bf52d2eae8ef829595e8c8975c1dfca8f3
+
+build:
+  script: python -m pip install . -vv
+  number: 0
```

You can add a noarch python here

```suggestion
  number: 0
  noarch: python
```